scripts/login.js

```diff
@@ -163,9 +163,15 @@ function retrieveUrl(location) {
   }
 }
 
-function setLandingTab(URL) {
+function setLandingTab(input) {
```

Comment:
It was changed to be more generic for anchor point or URL input - solution now is to say `href` :)